src/redux/sagas/transactions/createTransaction.ts

```diff
@@ -131,3 +132,18 @@ export function* waitForTxResult(channel: Channel<any>) {
     throw new Error('Transaction failed');
   }
 }
+
+export const createGroupTransaction = (
```

Comment:
I like this change but I think if you don't change all the sagas to use it (and pay attention to other PRs to make sure they're following this convention too), instead of improving things, it'll introduce inconsistencies that are really hard to resolve later (if I came across the two versions of `createGroupTransaction` in a couple months time, I'd need to dig through the code to understand if there's a difference or whether it's just refactoring-in-progress)
It might sound counterintuitive, but sometimes it's better to leave things as they are unless you're willing to make the change consistent